pkg/wallet/wallet.go

```diff
@@ -9,7 +9,7 @@ import (
 	"errors"
 
 	"github.com/massalabs/thyra/pkg/node/base58"
-	"golang.org/x/crypto/pbkdf2"
+	"github.com/xdg-go/pbkdf2"
```

Comment:
@jwmdev why changing pbkdf2 function package ?

Reply:
> @jwmdev why changing pbkdf2 function package ?

No reasons, I don't know what happened. Anyway, I see it resolved.